src/LaunchDarkly.EventSource/EventSource.cs

```diff
@@ -124,11 +125,6 @@ public EventSource(Configuration configuration)
         /// <exception cref="InvalidOperationException">The method was called after the connection <see cref="ReadyState"/> was Open or Connecting.</exception>
         public async Task StartAsync()
         {
-            if (_httpClient is null)
-            {
-                _httpClient = CreateHttpClient();
-            }
-
```

Comment:
I'm not sure why I was creating this lazily before. It's simpler to do it in the constructor and then if our parameters are invalid, we'll find out right away.